src/descriptors/power_system_structs.json

```diff
@@ -63,8 +63,8 @@
       {
         "name": "primemover",
         "comment": "PrimeMover Technology according to EIA 923",
-        "null_value": "nothing",
-        "data_type": "Union{Nothing, PrimeMovers}"
+        "null_value": "PrimeMover.HY::PrimeMovers",
```

Comment:
Does this work?  Why not just HY::PrimeMovers?  Or maybe PowerSystems.HY::PrimeMovers.